packages/inventory/src/Stocktake/DetailView/ContentArea/ContentArea.tsx

```diff
@@ -33,8 +32,6 @@ export const ContentArea: FC<{
   const { rows, onChangeSortBy, sortBy } = useStocktakeRows(isGrouped);
   const columns = useStocktakeColumns({ onChangeSortBy, sortBy });
 
-  if (!rows) return <DataTableSkeleton hasGroupBy={true} />;
```

Comment:
This was making the skeleton show up for brand new stocktakes after loading

Reply:
🤦